### PR TITLE
Version 1.4.3

### PR DESCRIPTION
**Pull request by sbaerlocher**
### Changed
Updated Syntax, Code and Documentation
### Fixed
Fix More Server by Windows
### Added
molecule tests

### CODE REVIEW

**Review by ndum — COMMENTED** [review]
Approved

**Review by FreeMinded — CHANGES_REQUESTED** [review]
Changelog.md line 14 should say
- Fixed Problem that only one NTP server could be specified.

File: tasks/main.yml [hunk 1]
```diff
@@ -1,38 +1,47 @@
...
+  include_vars: '{{ loop_vars }}'
   with_first_found:
     - files:
-        - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
-        - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
-        - "{{ ansible_distribution }}.yml"
-        - "{{ ansible_os_family }}.yml"
-        - "{{ ansible_system }}.yml"
-        - "defaults.yml"
+        - '{{ distribution }}-{{ distribution_verion }}.yml'
```
Comment (mleutenegger) [inline tasks/main.yml]:
```suggestion
        - '{{ distribution }}-{{ distribution_version }}.yml'
```

File: tasks/main.yml [hunk 2]
```diff
@@ -1,38 +1,47 @@
...
+  include_tasks: '{{ loop_distribution }}'
   with_first_found:
     - files:
-        - "{{ ansible_distribution }}-{{ ansible_distribution_version }}.yml"
-        - "{{ ansible_distribution }}-{{ ansible_distribution_major_version }}.yml"
-        - "{{ ansible_distribution }}.yml"
-        - "{{ ansible_os_family }}.yml"
-        - "{{ ansible_system }}.yml"
-        - "defaults.yml"
+        - '{{ distribution }}-{{ distribution_verion }}.yml'
```
Comment (mleutenegger) [inline tasks/main.yml]:
```suggestion
        - '{{ distribution }}-{{ distribution_version }}.yml'
```

File: tasks/main.yml [hunk 3]
```diff
@@ -1,38 +1,47 @@
...
+        - '{{ ansible_system }}.yml'
+        - 'defaults.yml'
       paths:
-        - "vars"
+        - 'vars'
   loop_control:
     loop_var: loop_vars
+  vars:
+    distribution: '{{ ansible_distribution }}'
+    distribution_verion: '{{ ansible_distribution_version }}'
```
Comment (mleutenegger) [inline tasks/main.yml]:
```suggestion
    distribution_version: '{{ ansible_distribution_version }}'
```

File: tasks/main.yml [hunk 4]
```diff
@@ -1,38 +1,47 @@
...
+        - '{{ ansible_system }}.yml'
+        - 'defaults.yml'
       paths:
-        - "distribution"
+        - 'distribution'
   loop_control:
     loop_var: loop_distribution
+  vars:
+    distribution: '{{ ansible_distribution }}'
+    distribution_verion: '{{ ansible_distribution_version }}'
```
Comment (mleutenegger) [inline tasks/main.yml]:
```suggestion
    distribution_version: '{{ ansible_distribution_version }}'
```

**Review by mleutenegger — APPROVED** [review]

**Review by FreeMinded — APPROVED** [review]

**Pull request merged**